src/Lykke.Service.Zcash.Api.Services/HealthService.cs

```diff
@@ -7,13 +9,30 @@ namespace Lykke.Service.Zcash.Api.Services
     // NOTE: See https://lykkex.atlassian.net/wiki/spaces/LKEWALLET/pages/35755585/Add+your+app+to+Monitoring
     public class HealthService : IHealthService
     {
-        public string GetHealthViolationMessage()
+        private readonly IBlockchainReader _blockchainReader;
+
+        public HealthService(IBlockchainReader blockchainReader)
+        {
+            _blockchainReader = blockchainReader;
+        }
+
+        public async Task<string> GetHealthViolationMessage()
         {
             // TODO: Check gathered health statistics, and return appropriate health violation message, or NULL if service hasn't critical errors
+
+            try
+            {
+                var info = await _blockchainReader.GetInfoAsync();
+            }
+            catch (Exception ex)
+            {
+                return ex.Message;
+            }
+
             return null;
         }
 
-        public IEnumerable<HealthIssue> GetHealthIssues()
+        public async Task<IEnumerable<HealthIssue>> GetHealthIssues()
```

Comment:
Why this method is async?

Reply:
Fixed. No need to be async anymore (was just for consistency).